File: 프로그래머스/lv0/120836. 순서쌍의 개수/순서쌍의 개수.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    int cnt =0;
    
    for(int i=1; i<=n; i++){
        if(n%i == 0){
            cnt++;
        }
    }
    answer = cnt;
    return answer;
}

File: 프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.java
class Solution {
    public int solution(int n) {
        int count = 0;

        for (int i = 1; i * i <= n; i++) {
            if (n % i == 0) {
                count++; // i는 n의 약수
                if (i != n / i) {
                    count++; // n/i도 n의 약수
                }
            }
        }

        return count;
    }
}
